File: 3342/3342.java
// bee 3342 - Keanu
// Alonso Martins
// 20/02/2024
import java.util.Scanner;

public class Main
{
	public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt(), pretas, brancas;
    
        pretas = (n * n) / 2;
        brancas = pretas + (n % 2);
        System.out.println(brancas+" casas brancas e "+pretas+" casas pretas");
	}
}

File: 3342/3342.c
// bee 3342 - Keanu
// Alonso Martins
// 20/02/2024
#include <stdio.h>

int main()
{
    int n, brancas = 0, pretas = 0;

    scanf("%d", &n);
    
    pretas = (n * n - (n % 2)) / 2;
    brancas = pretas + (n % 2);
    
    printf("%d casas brancas e %d casas pretas\n", brancas, pretas);
    
    return 0;
}
